applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/prj_release.conf

```diff
@@ -58,7 +58,7 @@ CONFIG_AVERAGE_CURRENT_CONSUMPTION=10000
 CONFIG_BOARD_SERIAL_BACKEND_CDC_ACM=n
 
 # Bootloader and MCUmgr (OTA DFU) Configuration
-CONFIG_MCUBOOT_IMAGE_VERSION="2.4.0+0"
+CONFIG_MCUBOOT_IMAGE_VERSION="2.4.1+0"
```

Comment:
Should we also find other occurrences of "2.4.0+0" and change it to "2.4.1+0" e.g. in Machine Learning sample.

Reply:
Can we do it in a separate PR?

Reply:
Sure